team-telnyx/telnyx-node
# DOTCOM-3584: convert resource files and tests to typescript

I have labeled some files as `important` for check when reviewing. Please use the file filter to omit the "excluded files" when reviewing


- [ ] resources
- [x] Jest base setup
- [x] Error

## Manual testing

- run `npm run lint`
- run `npm run build`
- check types in `types` folder
- check output in `dist` folder
- check README instructions on testing and typescript
- run `npm run test`
- see test report
